programs/drift/src/instructions/user.rs

```diff
@@ -3212,7 +3208,7 @@ pub struct InitializeSwiftUserOrders<'info> {
 pub struct ResizeSwiftUserOrders<'info> {
     #[account(
         mut,
-        seeds = [SWIFT_PDA_SEED.as_ref(), user.key().as_ref()],
+        seeds = [SWIFT_PDA_SEED.as_ref(), authority.key().as_ref()],
```

Comment:
not blocker: we should think about making this permissionless so mm can expand it